Design and log in/out

wanderlust-create opened the pull request:
Bootstrap Design 
-index
-dashboard
-search

and User
log_in, 
log_out functionality

markertmer commented:
1. I ran RSpec and there are five failing tests on this branch. Please make sure tests are all passing before opening a PR.
2. I'm seeing a lot of differences in the Gemfile.lock with what is currently working on fe_staging. It might be possible to remove the file from your commits, that way we can just use what we already have. Another thing to try might be to delete the Gemfile.lock and re-bundle.
3. In the Application Controller, I don't think `:require_user` and `:logged_in?` need to be added as helper methods. `:current_user` is there because it's called from each of the other two methods.
4. Why is there a `get` and `post` method for logout paths? Can we get just one of those to work? You may need to specify the method on one or the other if you've got logout buttons and links. If it's a button, I believe it does `post` by default, and links are `get`.

wanderlust-create commented:
1. Can you put a screenshot of the failing tests? They are all passing here except for the 1 skipped 'logged in user' index one.
2. I deleted the Gemfile and re-bundled when I switched to Bootstrap 4 not too long ago- I can look and see what the differences are.  
3. I need these for the header options and other places. I was getting error messages without them, so added them. 
4. This is what I needed to get log_in and log)out to work. Are they causing other problems?
